pkg/standalone/standalone.go

```diff
@@ -163,13 +197,18 @@ func Init(runtimeVersion string, dockerNetwork string, installLocation string, r
 
 	if s != nil {
 		s.Stop()
-		err = confirmContainerIsRunning(utils.CreateContainerName(DaprRedisContainerName, dockerNetwork))
+	}
+
+	print.SuccessStatusEvent(os.Stdout, msg)
+	print.InfoStatusEvent(os.Stdout, "%s binary has been installed.\n", daprRuntimeFilePrefix)
+	for _, container := range dockerContainerNames {
+		err = confirmContainerIsRunning(utils.CreateContainerName(container, dockerNetwork))
```

Comment:
This new code should be inside the if s != nil { } scope to maintain the old logic

Reply:
`confirmContainerIsRunning` method is to make sure that the container is running and contains only `docker` commands. I am not seeing a reason why we cannot confirm whether a container is running or not in windows environment. 
The spinner `s` is started only in a non-windows OS so that is why there is a nil check.